wlauto/resource_getters/standard.py

```diff
@@ -522,3 +540,16 @@ def get_owner_path(resource):
         return os.path.join(os.path.dirname(__base_filepath), 'common')
     else:
         return os.path.dirname(sys.modules[resource.owner.__module__].__file__)
+
+
+def get_abi_specific_path(resource, base_dir, extension, enforce=False):
+    # Note: Using device.get_abi() method instead of the property device.abi property
+    # The property is not yet set or available at this point in the execution flow
+    abi_dir = os.path.join(base_dir, extension, resource.owner.device.get_abi())
```

Comment:
Changed. Would it also make sense to rename this method to `get_platform_specific_path`, and the `kwargs` argument as `check_platform` (or something similar)?